app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,8 @@
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer-copyright text-center">© 2020 Copyright:
```

Comment:
Don't use hard coded `2020` it should be dynamic.

Reply:
cahnged